src/aap_eda/api/serializers/team.py

```diff
@@ -57,6 +57,9 @@ class Meta:
 
     def validate(self, data):
         self.validate_shared_resource()
+        validators.check_if_team_name_exists(
```

Comment:
@msmagnanijr When this resource gets managed in the Gateway does it follow the same constraints?